Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -808,12 +802,9 @@ public function getMethodDeclaredReturnType(string $className, string $methodNam
     public function getPropertyNamesByTag(string $className, string $tag): array
     {
         $className = $this->prepareClassReflectionForUsage($className);
-        if (!isset($this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES])) {
-            return [];
-        }
 
         $propertyNames = [];
-        foreach ($this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES] as $propertyName => $propertyData) {
+        foreach ($this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES] ?? [] as $propertyName => $propertyData) {
```

Comment:
Oh god, that is so much harder to read though, I wouldn't change that, shorter is not always better

Reply:
I'd say this combines short code with readability:
```suggestion
        $classProperties = $this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES] ?? []
        foreach ($classProperties as $propertyName => $propertyData) {
```

Reply:
jup adjusted it :)